bundles/paikkatietoikkuna/coordinatetransformation/components/Lableled.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
Is this file used anywhere? The name has an extra l but I don't see an import for this anywhere?